app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ArticlesController < ApplicationController
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+    if @article.save
+      redirect_to @article, notice: "Article was successfully created."
```

Comment:
Can we move this string into a locale file? It's a good practice to prepare ourselves for internationalisation (i18n) from the beginning. It also separates our content from our behaviour.

(ditto throughout)